run_thread_switcher_aida64.bat

```diff
@@ -0,0 +1,7 @@
+pushd %~dp0
+
+cp settings_aida64.py settings.py
```

Comment:
```suggestion
copy settings_aida64.py settings.py
```
cp is not available for me on cmd, so the .bat didn't work

Reply:
Good catch. I will change this.

It looks like `copy` does not overwrite by default. I will add the `/Y` flag so that it does.